guides/main_concepts/indexes.md

```diff
@@ -36,66 +35,68 @@ The `uid` cannot be changed.
 
-On top of that, you can add your own rule to the ranking rules. For example, you can rank your movies by release date or per popularity. It's also possible to do both. And so on, depending on your available data and your user's needs.
+On top of that, you can add your custom rules to the ranking rules. For example, you may want to rank your movies either by release date or popularity, or both and so on. Rules are customizable so the results meet your user's needs as close as possible.
 
-[More information about ranking rules](/guides/main_concepts/relevancy.md)
+[Learn more about ranking rules](/guides/main_concepts/relevancy.md)
 
-## Synonyms and stop-words
+## Synonyms
 
-An index can contain a set of synonyms. If multiples words have an equivalent meaning in your dataset, you can decide to create a synonym for theses words. The search engine will give the same search results for any search with one of the associated words as a search query. The synonyms are linked to the given index, and they will not apply to any other index on the same MeiliSearch instance.
+A set of synonyms can be defined for an index. In your dataset, you may decide to create synonyms for words which have the same meaning. Even though they look different, they should be treated similarly. If either of the associated words is searched, the same results shall be displayed. Since synonyms are linked to a given index, they won't apply to any other index on the same MeiliSearch instance.
 
-[More information about synonyms](/guides/advanced_guides/synonyms.md)
+[Learn more about synonyms](/guides/advanced_guides/synonyms.md)
 
-An index can contain a list of stop-words. Those words will be ignored in documents and search queries. Typically those words could be redundant words of your chosen language, like `the` or `of` in English.
-By adding those words in the stop-words list, you avoid having documents considered highly relevant because of the recurrence of one of those words in a document.
+## Stop words
 
-For example, on the following search query: `the great gatsby`, if the presence of the word `the` in a film review should not make the review more relevant. By adding `the` to the stop-word list, you avoid searching in documents containing only this stop word.
+Sometimes you may want to ignore certain words in documents and search queries. To do so, a set of stop words can be defined for an index. Unless you actually need them, some words neither add semantic value nor context. Besides, they are often too frequent (i.e. `the` or `of` in English).
+By adding words to a stop words list, these specific terms will be excluded from your search. It will avoid documents being considered highly relevant because of the high frequency of some words in a corpus.
```

Comment:
So i made the same assumption as you just did, but a document containing multiple time a search query will not be considered more relevant than another that has those search query words only once.

But if `The` is present in the `title` (which is the most important attribute) and also at the start of `title` (see position in the ranking rules) the document will be considered highly relevant. 

Furthermore, `the` being so popular, when searching for `the` Meilisearch will have to first gather all documents that contains `the` and then start sorting them. Which could be 90% of your database. Which makes the search slower.
```suggestion
By adding words to a stop words list, these specific terms will be excluded from your search. It will avoid documents being considered highly relevant because of the presence of some words in an important attribute(/guides/main_concepts/relevancy.md#_4-attribute) or in a good [position](/guides/main_concepts/relevancy.md#ranking-rules).
This will also greatly improve the search time because all the documents containing only those words will not be used in the sorting of documents.
```

Reply:
Oh! Right, I get it (attribute importance)! :o Thanks!